python/tests/integration/test_methods.py

```diff
@@ -357,6 +379,47 @@ def test_crud_look(sdk: mtds.Looker40SDK, looks):
         assert not look.deleted
 
 
+def test_png_svg_downloads(sdk: mtds.Looker40SDK):
+    """content_thumbnail() should return a binary or string response based on the specified format."""
+    looks = sdk.search_looks(limit=1)
+    _id: str
+    if looks:
+        _type = "look"
+        _id = str(looks[0].id)
```

Comment:
maybe `resource_id` and `resource_type` ?
If you want to avoid keyword collision then the convention is trailing underscore (`id_` and `type_`):
> single_trailing_underscore_: used by convention to avoid conflicts with Python keyword

from [PEP8](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)

Reply:
`resource_id` and `resource_type` it is